straxen/plugins/peak_processing.py

```diff
@@ -497,6 +497,7 @@ def compute_shadow(self, peaks, current_peak):
         return result
 
     @staticmethod
+    @np.errstate(invalid='ignore')
```

Comment:
Hi @dachengx , thanks, this will definitely solve the error. In case we use `@np.errstate(invalid='ignore')` I don't think a version bump (and implied reprocessing) is really necessary right: the code will produce exactly the same data. Do you agree?

Reply:
I agree. The `@np.errstate` does not influence any results.

Reply:
I just revert the version bump of `PeakShadow`.